examples/flight-tracker/tracker-app-config.yaml

```diff
@@ -79,6 +92,11 @@ spec:
             metadata:
               name: tracker-ingress
             spec:
-                backend:
-                  serviceName: web-ui
-                  servicePort: 8080
\ No newline at end of file
+              rules:
+                - host: x.oam.dev
+                  http:
+                    paths:
+                      - backend:
+                          serviceName: web-ui
+                          servicePort: 8080
+                        path: /
```

Comment:
Users don't own this domain and could not view it if we setting like this. Please keep the original way, thanks.

Reply:
ok, i will keep the original way.